src/main/java/com/capitalone/dashboard/webhook/github/GitHubPullRequestV3.java

```diff
@@ -546,4 +547,21 @@ private boolean isValidEvent(String action) {
         return validPullRequestEvents.contains(PullRequestEvent.fromString(action));
     }
 
+    // Add pull number to commits in the PR if they don't have one
+    private void updateCommitsWithPullNumber(GitRequest pull) {
+        Commit commit;
+        for(Commit c : pull.getCommits()) {
+            commit = commitRepository.findByCollectorItemIdAndScmRevisionNumber(pull.getCollectorItemId(), c.getScmRevisionNumber());
```

Comment:
@chzhanpeng  is it possible to have multiple commits with same scmRevisionNumber.